Makefile

```diff
@@ -35,17 +36,23 @@ bin:
 driver: | bin
 	CGO_ENABLED=0 go build -ldflags ${LDFLAGS} -o bin/ibm-powervs-block-csi-driver ./cmd/
 
+.PHONY: node-update-controller
+node-update-controller: | bin
+	CGO_ENABLED=0 go build -ldflags ${LDFLAGS} -o bin/node-update-controller ./adhoc-controllers/
+
 .PHONY: test
 test:
 	go test -v -race ./cmd/... ./pkg/...
 
 .PHONY: image
 image:
 	docker build -t $(REGISTRY)/$(IMG):$(TAG) . --target debian-base
+	docker build -t $(REGISTRY)/$(IMG_NODE_UPDATE_CONTROLLER):$(TAG) . --target debian-base-node-update-controller
 
 .PHONY: push
 push:
 	docker push $(REGISTRY)/$(IMG):$(TAG)
+	docker push $(REGISTRY)/$(IMG_NODE_UPDATE_CONTROLLER):$(TAG)
```

Comment:
No need to build node-update-controller every time when csi driver is built.
Create separate target for pushing node-update-controller image.